docs/src/dataDisplay/table/tableSampleSize.jsx

```diff
@@ -1,108 +1,9 @@
-                        <Table.HeaderCell>Header 2</Table.HeaderCell>
-                        <Table.HeaderCell>Header 3</Table.HeaderCell>
-                        <Table.HeaderCell>Header 4</Table.HeaderCell>
-                    </Table.Row>
-                </Table.Header>
-
-                <Table.Body>
-                    <Table.Row>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                    </Table.Row>
-
-                    <Table.Row>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                    </Table.Row>
-
-                    <Table.Row>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                    </Table.Row>
-                </Table.Body>
-            </Table>
-        );
-    }
-}`;
-
-const sizeMediumTableSample = `import React from 'react';
 import { Table } from 'react-cm-ui';
 
-export default class SizeMediumTableSample extends React.Component {
-    render() {
-        return (
-            <Table basic size="medium">
-                <Table.Header>
-                    <Table.Row>
-                        <Table.HeaderCell>Header 1</Table.HeaderCell>
-                        <Table.HeaderCell>Header 2</Table.HeaderCell>
-                        <Table.HeaderCell>Header 3</Table.HeaderCell>
-                        <Table.HeaderCell>Header 4</Table.HeaderCell>
-                    </Table.Row>
-                </Table.Header>
-
-                <Table.Body>
-                    <Table.Row>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                    </Table.Row>
-
-                    <Table.Row>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                    </Table.Row>
-
-                    <Table.Row>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                        <Table.Cell>cell</Table.Cell>
-                    </Table.Row>
-                </Table.Body>
-            </Table>
-        );
-    }
-}`;
-
 function TableSampleSize() {
     return (
         <React.Fragment>
```

Comment:
I don't think we need `<React.Fragment>` anymore if we're returning a single `<Table>` ... right?